debian/control

```diff
@@ -43,6 +43,7 @@ Depends: cloud-guest-utils | cloud-utils,
          ${python3:Depends}
 Recommends: eatmydata, gdisk, gnupg, python3-apt, software-properties-common
 Suggests: openssh-server, ssh-import-id
+Breaks: python3-minimal (<< 3.12.3-0ubuntu2~)
```

Comment:
Note, this is sync'd from direct release of 24.2-0ubuntu1~24.04.2 to the distribution packaging branches for ubuntu/noble-proposed/updates